man/systemd.netdev.xml

```diff
@@ -571,6 +571,16 @@
           <xi:include href="version-info.xml" xpointer="v248"/>
         </listitem>
       </varlistentry>
+     <varlistentry>
+        <term><varname>BroadcastThreshold=</varname></term>
+        <listitem>
+          <para>Specifies the broadcast queueing threshold on macvlan devices. This controls which multicast
+          packets will be processed in a workqueue instead of inline. Setting this to "no" disables broadcast
+          queueing altogether. An integer in the range -2147483648…2147483646. Defaults to unset.</para>
```

Comment:
All negative values should have the same effect, that is, disable broadcasting.

At least there are no place where the kernel or `ip` command refuse `INT32_MAX`.

> No I tested it's allowed till `2147483646`

How do you test?

Reply:
via a .netdev file
```
[NetDev]
Name=mac0
Kind=macvlan

[MACVLAN]
Mode=bridge
BroadcastQueueThreshold=-2147483648

```